sites/bizbash/config/site.js

```diff
@@ -79,7 +79,8 @@ module.exports = {
           { href: '/magazine/5b2a4e6c0305572b008b45ae', label: 'Our Publications' },
           { href: 'http://www.masterplanneronline.com', label: 'Masterplanner', target: '_blank' },
           { href: 'https://jobs.bizbash.com', label: 'Careers', target: '_blank' },
-        ],
+          { href: '/production-strategy/virtual-events', label: 'Virtual Events' },
+      ],
```

Comment:
This is causing a lint error... the bracket should be indented by two more spaces. :)

Reply:
Yea that would do it 😅 just fixed!